bin/webpack-dev-server.js

```diff
@@ -19,16 +19,8 @@ const port = config.get('webpackServerPort');
 const compiler = webpack(webpackDevConfig);
 
 const serverOptions = {
-  contentBase: `http://${host}:${port}`,
   headers: { 'Access-Control-Allow-Origin': '*' },
-  hot: true,
-  inline: true,
-  lazy: false,
-  noInfo: true,
-  progress: true,
   publicPath: webpackDevConfig.output.publicPath,
-  quiet: true,
-  stats: { colors: true },
```

Comment:
I saw you comment about us losing colors. In an earlier PR related to webpack-cli I needed to make some changes regarding `stats` and `colors`, so I'm surprised that there's nothing we can replace this with that will work. It's not a big deal, but might be worth doing a bit more digging to see if there's an option that will allow us to keep the colors.